projects/arlas-toolkit/src/lib/components/histogram-tooltip-overlay/histogram-tooltip-overlay.component.css

```diff
@@ -11,50 +11,84 @@
   min-width: 100px;
   box-shadow: 0 2px 4px -1px rgba(0, 0, 0, 0.2), 0 4px 5px 0 rgba(0, 0, 0, 0.14),
     0 1px 10px 0 rgba(0, 0, 0, 0.12);
+  display: grid;
+  grid-template-columns: minmax(60px, auto) 1fr;
+  grid-template-rows: repeat(3, minmax(10px, auto));
+  gap: 5px 10px;
 }
 
-.axis {
-  color: #999;
-  padding-right: 5px;
-  padding-left: 5px;
+.tooltip__header {
+  grid-column: 1/span 2;
+  grid-row: 1;
 }
 
-.xlabel {
-  color: #333;
+.tooltip__xLabel {
+  grid-column: 1;
+  grid-row: 2;
 }
-.title {
-  font-weight: bold;
-  color: #333;
+
+.tooltip__xValue {
+  grid-column: 2;
+  grid-row: 2;
 }
 
-.interval {
-  display: flex;
-  margin-left: 10px;
-  align-items: center;
+.tooltip__yLabel {
+  grid-column: 1;
+  grid-row: 3;
+}
+
+.tooltip__yValue {
+  grid-column: 2;
+  grid-row: 3;
+}
+.tooltip__label {
+  text-align: justify;
+}
+.header {
+  text-align: center;
+  padding: 0 5px;
+}
+
+.interval-text, .tooltip__label {
   font-size: 0.9em;
   color: #999;
 }
 
-.xline {
-  align-items: center;
+.axis-label {
+  display: inline-block;
 }
-.xline,
-.yline {
+
+.value-label {
+  text-align: end;
   display: flex;
-  justify-content: space-between;
-  min-width: 100px;
-  margin-bottom: 5px;
+  justify-content: end;
+  gap:4px;
 }
 
-.yline {
-  margin-top: 2px;
+.label {
+  color: #999;
+  padding-right: 5px;
+  padding-left: 5px;
+  text-align: start;
+  max-width: 200px;
 }
 
-.chip_count {
+.header {
   display: flex;
-  justify-content: space-between;
-  float: left;
-  text-align: right;
+  align-items: center;
+}
+
+.header-title {
+  font-weight: bold;
+  color: #333;
+}
+
+.interval {
+  display: flex;
+  margin-left: 10px;
+  align-items: center;
+  font-size: 0.9em;
```

Comment:
I suggest you use sizes in px as it is what we mostly used in the rest of the app. 12px would be the closest ?

Reply:
ok i will change. Its more 14 px.